code/modules/mob/living/carbon/alien/humanoid/humanoid.dm

```diff
@@ -210,19 +210,25 @@
 				help_shake_act(M)
 			else
 				if(M.health >= -75.0)
-					if(((M.head && M.head.flags & 4) || ((M.wear_mask && !( M.wear_mask.flags & 32 )) || ((head && head.flags & 4) || (wear_mask && !( wear_mask.flags & 32))))))
+					if(((M.head && M.head.flags & 4) || ((M.wear_mask && !( M.wear_mask.flags & 32 )) || ((head && head.flags & 4) || (wear_mask && !( wear_mask.flags & 32)))))) //Holy fucking christ why
```

Comment:
Honestly is there any reason why you can give CPR to aliens?
I just copypasted the new CPR code expecting it not to work and was fully prepared to just remove it all when it didn't work
Sadly, it did work